### PR TITLE
Fix debounce for Find entries

### PR DESCRIPTION
**Pull request by wshands**
Addresses:
dockstore/dockstore#3418
https://ucsc-cgl.atlassian.net/browse/DOCK-1327

### CODE REVIEW

File: src/app/home-page/widget/filtered-list.ts [hunk 1]
```diff
@@ -14,17 +15,25 @@ export abstract class FilteredList extends Base implements OnInit {
...
   constructor(protected userQuery: UserQuery, protected entriesService: EntriesService, protected usersService: UsersService) {
     super();
   }
 
   ngOnInit() {
     this.getMyList();
+
+    this.subject.pipe(
+      // formDebounceTime is set to 250 which is not long enough
+      debounceTime(500)
```
Comment (garyluu) [inline src/app/home-page/widget/filtered-list.ts:29]: also need to unsubscribe and https://www.learnrxjs.io/learn-rxjs/operators/filtering/distinctuntilchanged

File: src/app/home-page/widget/filtered-list.ts [hunk 2]
```diff
@@ -14,17 +15,27 @@ export abstract class FilteredList extends Base implements OnInit {
...
 
   ngOnInit() {
     this.getMyList();
+
+    this.subject.pipe(
+      // The usual argument formDebounceTime is set to 250 which is not long enough
+      debounceTime(500),
+      distinctUntilChanged(),
+      takeUntil(this.ngUnsubscribe)
+  ).subscribe(filterText => {
```
Comment (coverbeck) [inline src/app/home-page/widget/filtered-list.ts:32]: Indentation seems off for this line.

File: src/app/home-page/widget/filtered-list.ts [hunk 3]
```diff
@@ -1,7 +1,8 @@
-import { OnInit } from '@angular/core';
+import {OnDestroy, OnInit} from '@angular/core';
```
Comment (coverbeck) [inline src/app/home-page/widget/filtered-list.ts:1]: LGTM flagged this, but I believe it's unused.

Comment (wshands) [reply src/app/home-page/widget/filtered-list.ts]: removed unused import

File: src/app/home-page/widget/filtered-list.ts (hunk 2 above)
Comment (garyluu) [reply src/app/home-page/widget/filtered-list.ts]: Hmm...prettier should've automatically ran.  Strange it didn't.

Comment (wshands) [reply src/app/home-page/widget/filtered-list.ts]: fixed indentation

**Pull request merged**